src/components/TasksSubList/TasksSubList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Typography } from '@material-ui/core';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { colors } from 'styles/theme';
+import TaskCheckbox from '../TaskCheckbox';
+import { TasksSubListProps } from './types';
+
+const useStyles = makeStyles({
+  header: {
+    margin: '1rem 0',
+    color: colors.custom.lightText,
+  },
+});
+
+const TasksSubList: React.FC<TasksSubListProps> = ({
+  tasks,
+  count,
+  title,
+  onChange,
+}) => {
+  const { header } = useStyles();
+
+  return (
+    <Box>
+      <Typography className={header} component='h4' variant='h4'>
+        {title} ({count || 0})
+      </Typography>
+      <Box>
```

Comment:
is this box needed?

Reply:
Yes - To keep html structure consistent, `TaskCheckbox` components should be wrapped in a container. It's similar to `<ul>` and `<li>` - list items should be wrapped with list container